README.md

```diff
@@ -29,8 +29,21 @@ cargo install --git https://github.com/paritytech/polkadot --tag <version> polka
 
 ### Build from Source
 
-If you'd like to build from source, first install Rust. You may need to add Cargo's bin directory
-to your PATH environment variable. Restarting your computer will do this for you automatically.
+If you'd like to build from source, follow these instructions.
+
+#### Quick Setup
+
+Install all the required dependencies with a single command. (Be patient, this can take up to 30
+minutes):
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
```

Comment:
Please no. Please let us kill this script finally...

Reply:
Can you provide more some context than that, Basti?

Reply:
This script was created 2018 for a demo. 

There are multiple problems with this script.

1. It installs the `substrate` node executable if you forget `--fast`, which is not required for Polkadot.
2. It only supports a small list of operating system/distributions.
3. I think it is much better to name the dependencies to make it easy to install them. I would expect that people who are using their computer, are able to install dependencies.

Reply:
Makes sense...thanks for the info. The PR has been updated.